.travis.yml

```diff
@@ -10,6 +10,22 @@ matrix:
     env: COVER=1
   - python: '2.7'
     env: CROSSDOCK=1
+  - python: '2.7.10'
+    env: CROSSDOCK=1
+  - python: '2.7.11'
+    env: CROSSDOCK=1
+  - python: '2.7.12'
+    env: CROSSDOCK=1
+  - python: '2.7.13'
+    env: CROSSDOCK=1
+  - python: '3.3'
+    env: CROSSDOCK=1
+  - python: '3.4'
+    env: CROSSDOCK=1
+  - python: '3.5'
+    env: CROSSDOCK=1
+  - python: '3.6'
```

Comment:
Jaeger currently doesn't work with 3.x, what's the reason for adding it?

Also, what's the reason for adding so many 2.7 minor versions, is there really any difference? It will make the tests run forever.

Reply:
Ah - I was under the impression that we were supporting 3.x as well. Will remove.

All these 2.7.x version are currently in use at Uber. My reasoning for this was that the tests in this repo does not seems to take very long, so it is better to prevent future surprises.